app/models/host/managed.rb

```diff
@@ -728,7 +728,7 @@ def explicit_pxe_loader
   end
 
   def pxe_loader
```

Comment:
I'm a fan of understanding what a method is supposed to return.
```suggestion
  # @return [String] The name of the PXE loader to be used or empty string if none
  def pxe_loader
```